arlas-tests/src/test/java/io/arlas/server/rest/collections/CollectionServiceIT.java

```diff
@@ -122,7 +122,7 @@ public void testImportExportCollections() throws Exception {
         given().multiPart("file", jsonExport)
                 .when().post(arlasPath + "collections/_import")
                 .then().statusCode(200)
-                .body("collection_name", everyItem(equalTo(COLLECTION_NAME)));
+                .body("collection_name", allOf(hasItem(COLLECTION_NAME_ACTOR),hasItem(COLLECTION_NAME)));
```

Comment:
You'd better use directly `hasItems` (cf. http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#hasItems(T...) )